docs/app/assets/js/search-worker.js

```diff
@@ -1,6 +1,7 @@
-"use strict";
-/* jshint browser: true */
-/* global importScripts, onmessage: true, postMessage, lunr */
+'use strict';
+
+/* eslint-env worker */
+/* global importScripts, lunr */
```

Comment:
>
> Actually, is there any reason why ignore assets?
>
The reason is docs/app/assets/js/angular-bootstrap/** which contains code
pasted from AngularUI. I changed the ignore rules.